src/main/groovy/life/qbic/datamodel/dtos/business/AffiliationCategory.groovy

```diff
@@ -0,0 +1,48 @@
+package life.qbic.datamodel.dtos.business
+
+/**
+ * We define three major business affiliation categories here, which
+ * means that we decide between
+ *  - internal: An affiliation we consider as within the University of Tübingen or University
+ *  Hospital of Tübingen
+ *  - external academic: An outside affiliation but an academic institution (public research
+ *  institutions)
+ *  - external: An outside affiliation but not academic (i.e. private sector, companies, etc)
+ */
+enum AffiliationCategory {
+
+  INTERNAL("internal"),
+  EXTERNAL_ACADEMIC("external academic"),
```

Comment:
In the architecture diagram we specified this value as ExternalAcademics, I'd agree that the s can be removed and will update the diagram accordingly

Reply:
a nice spot